srcpkgs/Solaar/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'Solaar'
 pkgname=Solaar
-version=1.1.5
-revision=2
+version=1.1.7
+revision=1
+wrksrc="solaar-${version}"
```

Comment:
wrksrc is not necessary anymore

Reply:
Huh, is that a thing now?! Have to read the updated manual I guess.